strings/strings.c

```diff
@@ -9,7 +9,12 @@
 */
 int string_length(char s[])
 {
+    int i, length = 0;
 
+    for(i = 0; s[i] != 0; i++) {
+        length++;
+    }
+    return length;
```

Comment:
Interesting implementation here! I like your unique structure of your `for` loop